composer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "buildempire/apartment",
+  "description": "Multi-tenancy for Lumen using PostgreSQL with Eloquent",
+  "keywords" : ["lumen", "postgres", "multitenancy"],
+  "type": "library",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Cameron Smith",
+      "email": "contact@buildempire.co.uk"
+    }
+  ],
+  "minimum-stability": "dev",
+  "require": {
+    "php": ">=5.6.4",
+    "laravel/lumen-framework": "5.4.*"
+  },
+  "version": "1.0.0"
```

Comment:
@uk-casmith please transpose some of the discussion (key points will be fine) onto the comments so that others are aware of what was discussed. :)